plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -414,93 +417,93 @@ Setting a color to black (#000000) will make it invisible.""")
+        put(Blocks.DEEPSLATE, 0x575759);
+        put(Blocks.DETECTOR_RAIL, 0x000000);
         put(Blocks.DIAMOND_BLOCK, 0x65EFE5);
         put(Blocks.DIAMOND_ORE, 0x788F8F);
-        put(Blocks.DIORITE, 0xBDBDBD);
         put(Blocks.DIORITE_SLAB, 0xBDBDBD);
         put(Blocks.DIORITE_STAIRS, 0xBDBDBD);
         put(Blocks.DIORITE_WALL, 0xBDBDBD);
-        put(Blocks.DIRT, 0x866043);
+        put(Blocks.DIORITE, 0xBDBDBD);
         put(Blocks.DIRT_PATH, 0x947A41);
+        put(Blocks.DIRT, 0x866043);
         put(Blocks.DISPENSER, 0x727171);
         put(Blocks.DRAGON_EGG, 0x0D0910);
-        put(Blocks.DRAGON_HEAD, 0x1D1A1E);
-        put(Blocks.DRAGON_WALL_HEAD, 0x1D1A1E);
+        put(Blocks.DRAGON_HEAD, 0x000000);
+        put(Blocks.DRAGON_WALL_HEAD, 0x000000);
         put(Blocks.DRIED_KELP_BLOCK, 0x343C28);
         put(Blocks.DRIPSTONE_BLOCK, 0x866B5C);
         put(Blocks.DROPPER, 0x727171);
         put(Blocks.EMERALD_BLOCK, 0x2BCD5A);
         put(Blocks.EMERALD_ORE, 0x6A8972);
         put(Blocks.ENCHANTING_TABLE, 0x7E3F4B);
         put(Blocks.END_GATEWAY, 0x030303);
-        put(Blocks.END_PORTAL, 0x030303);
         put(Blocks.END_PORTAL_FRAME, 0x5A755E);
-        put(Blocks.END_ROD, 0xA4928C);
-        put(Blocks.END_STONE, 0xDBDF9E);
+        put(Blocks.END_PORTAL, 0x030303);
+        put(Blocks.END_ROD, 0xC8BFB5);
         put(Blocks.END_STONE_BRICK_SLAB, 0xDBE0A2);
         put(Blocks.END_STONE_BRICK_STAIRS, 0xDBE0A2);
         put(Blocks.END_STONE_BRICK_WALL, 0xDBE0A2);
         put(Blocks.END_STONE_BRICKS, 0xDBE0A2);
+        put(Blocks.END_STONE, 0xDBDF9E);
         put(Blocks.ENDER_CHEST, 0x2C3D3F);
         put(Blocks.EXPOSED_COPPER, 0xA17E68);
-        put(Blocks.EXPOSED_CUT_COPPER, 0x9B7A65);
         put(Blocks.EXPOSED_CUT_COPPER_SLAB, 0x9B7A65);
         put(Blocks.EXPOSED_CUT_COPPER_STAIRS, 0x9B7A65);
-        put(Blocks.FARMLAND, 0x8E6646);
+        put(Blocks.EXPOSED_CUT_COPPER, 0x9B7A65);
         put(Blocks.FERN, 0x000000);
-        put(Blocks.FIRE, 0xD48C35);
-        put(Blocks.FIRE_CORAL, 0xA6252F);
         put(Blocks.FIRE_CORAL_BLOCK, 0xA4232F);
         put(Blocks.FIRE_CORAL_FAN, 0x9F232E);
         put(Blocks.FIRE_CORAL_WALL_FAN, 0x9F232E);
+        put(Blocks.FIRE_CORAL, 0xA6252F);
+        put(Blocks.FIRE, 0xD48C35);
         put(Blocks.FLETCHING_TABLE, 0xC6B687);
-        put(Blocks.FLOWER_POT, 0x5A3227);
-        put(Blocks.FLOWERING_AZALEA, 0x9D5CAB);
-        put(Blocks.FLOWERING_AZALEA_LEAVES, 0x9D5CAB);
+        put(Blocks.FLOWER_POT, 0x000000);
+        put(Blocks.FLOWERING_AZALEA_LEAVES, 0x646F3D);
+        put(Blocks.FLOWERING_AZALEA, 0x717A41);
         put(Blocks.FROGSPAWN, 0x6A5B52);
         put(Blocks.FROSTED_ICE, 0x8CB5FD);
         put(Blocks.FURNACE, 0x727171);
         put(Blocks.GILDED_BLACKSTONE, 0x382B27);
+        put(Blocks.GLASS_PANE, 0x000000);
         put(Blocks.GLASS, 0xB0D6DB);
-        put(Blocks.GLASS_PANE, 0xB0D6DB);
+        put(Blocks.GLOW_ITEM_FRAME, 0xA17246);
```

Comment:
Item frames should be invisible since they are mostly used in an upright position, which would make them not visible from a top down view (like the banners)